pkgs/applications/misc/kjv/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "kjv";
+  version = "20190311";
+
+  src = fetchFromGitHub {
+    owner = "LukeSmithxyz";
+    repo = "${name}";
```

`repo = name` works just fine here

```suggestion
    repo = name;
```